transcode_system_migration.sql

```diff
@@ -505,6 +547,33 @@ UPDATE [dbo].[media_format]
 SET mapped_to_format_id=-1
 WHERE media_formatid IN (SELECT media_format_id FROM @source_copy_media_format_ids);
 
+update Formats
+set NoSecurityWhenInChannelFolderIds = coalesce((select json_arrayagg(t.LayoutfolderId)
+                                                 from (select distinct LPD.LayoutfolderId
+                                                       from dz_profileformat pf
+                                                                join dz_profile p on pf.dz_profileid = p.dz_profileid
+                                                                join Layoutfolder_Profile_Destination LPD
+                                                                     on p.dz_profileid = LPD.Dz_ProfileId
+                                                                join digitranscode_destination maybe_storage_manager_destination
+                                                                     on LPD.DestinationId =
+                                                                        maybe_storage_manager_destination.digitranscode_destinationid
+                                                                left join digitranscode_destination maybe_specific_destination
+                                                                          on maybe_specific_destination.StorageManagerId =
+                                                                             maybe_storage_manager_destination.digitranscode_destinationid
+                                                       where pf.media_formatid in
+                                                             (SELECT media_format_id FROM @source_copy_media_format_ids)
+                                                         and (maybe_storage_manager_destination.LaxSecurity = 1 or
+                                                              maybe_specific_destination.LaxSecurity = 1)) as t), '[]'),
+    PreGenerateForChannelFolderIds   = coalesce(
+            (select json_arrayagg(t.LayoutfolderId)
+             from (select distinct LPD.LayoutfolderId
+                   from dz_profileformat pf
+                            join dz_profile p on pf.dz_profileid = p.dz_profileid
+                            join Layoutfolder_Profile_Destination LPD on p.dz_profileid = LPD.Dz_ProfileId
+                   where pf.media_formatid in (SELECT media_format_id FROM @source_copy_media_format_ids)) as t)
```

Comment:
Do we need to use `PreGenerateForChannelFolderIds` for the `Original` format?
I don't really see the use-case for it, but we of course need to test that `NoSecurityWhenInChannelFolderIds` works without using the `PreGenerateForChannelFolderIds` option.

Reply:
It was mostly a consistency thing to be honest.